src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="day-1 calendar__day calendar__day--start-day--mon"></div>
```

Comment:
it should be calendar modifiers:
![image](https://user-images.githubusercontent.com/86923089/194704667-7303262a-bde6-43c2-aa5b-cb71132d9147.png)